pages/ksphere/kommander/latest/clusters/attach-cluster/index.md

```diff
@@ -117,10 +76,57 @@ users:
 EOF
 ```
 
-This produces a file in your current working directory called `kommander-cluster-admin-config`. The contents of this file can uploaded to Kommander to import the cluster using the dedicated service account.
+This produces a file in your current working directory called `kommander-cluster-admin-config`. The content of this file is what you should use in Kommander to attach the cluster.
```

Comment:
```suggestion
This produces a file in your current working directory called `kommander-cluster-admin-config`. The contents of this file are what you should use in Kommander to attach the cluster.
```

IIUC then it is `content` if you are talking about something specific and `contents`  if you mean all of what is contained. Native speakers please chime in :)